src/biotite/sequence/graphics/alignment.py

```diff
@@ -408,7 +405,10 @@ def plot_alignment(axes, alignment, symbol_plotter, symbols_per_line=50,
     spacing : float, optional
         The spacing between the alignment lines. 1.0 means that the size
         is equal to the size of a symbol box.
-    
+    symbol_spacing : int, optional
+        А space is placed between each number of elements desired
+        by variable (github issue #360)
```

Comment:
GitHub issues are not referenced in docstrings in Biotite, at least no in the `Parameters` section.

```suggestion
        by variable.
```